modules/spark/src/test/scala/pureconfig/module/spark/sql/SparkSqlTest.scala

```diff
@@ -0,0 +1,37 @@
+package pureconfig.module.spark.sql
+
+import com.typesafe.config.ConfigValueFactory
+import org.apache.spark.sql.types._
+
+import pureconfig.BaseSuite
+import pureconfig.error.CannotConvert
+import pureconfig.generic.auto._
+import pureconfig.module.spark.sql.arbitrary._
+
+class SparkSqlTest extends BaseSuite {
```

Comment:
Just for consistency with all other tests in PureConfig.

```suggestion
class SparkSqlSuite extends BaseSuite {
```